audition/utils.py

```diff
@@ -1,2 +1,17 @@
+from catwalk.evaluation import ModelEvaluator
+import logging
+
+
 def str_in_sql(values):
     return ','.join(map(lambda x: "'{}'".format(x), values))
+
+
+def greater_is_better(metric):
+    if metric in ModelEvaluator.available_metrics:
+        return ModelEvaluator.available_metrics[metric].greater_is_better
+    else:
+        logging.warning(
+            'Metric %s not found in available metrics, assuming greater is better',
+            metric
+        )
```

Comment:
I might actually be inclined to raise an exception here rather than just issue a warning and make an assumption -- we should probably allow some way of including custom metrics as we do in catwalk, but regardless should continue to enforce the constraint that metrics always have a `greater_is_better` attribute